controllers/CtrlFeeds.php

```diff
@@ -2,24 +2,78 @@
 
 namespace Controllers;
 
+use Model\FeedModel;
+use Model\NewsModel;
+use Controllers\CtrlNews;
 use MVC\Router;
 
 class CtrlFeeds{
 
-    public static function getAllFeeds(){
-    }
+  
+    public static function registerFeed($router ){
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+            
+            $urls = $_POST['url'];
+            $feeds = recuperarFeeds($urls);  
+            $feeddb = new FeedModel;
+            $feeddb->deleteAll();
+
+            
+            foreach($feeds as $feed){
+                
+                $alerts = [];
+
+                $feeddb->feedName = $feed->get_title();
+                $feeddb->feedUrl = $feed->get_permalink();
+                if ($feed->get_image_url() == null) {
+                    $feeddb->feedImageUrl = 'no hay imagen disponible';
+                }else {
+                    $feeddb->feedImageUrl = $feed->get_image_url();
+                }
```

Comment:
```suggestion
                $feeddb->feedImageUrl = $feed->get_image_url();
```
Este metodo retorna la URL o null en caso de no existir, tenemos pensado que en caso de null [añadimos una imagen por default de rss](
https://upload.wikimedia.org/wikipedia/commons/thumb/4/43/Feed-icon.svg/800px-Feed-icon.svg.png)

Reply:
Puse la imagen que me pusiste como ejemplo, porque este es el unico lugar donde ingresamos datos, entonces es el unico momento de la ejecución donde se puede ingresar